### PR TITLE
Add some simple retry to service rio commands

### PR DESCRIPTION
**Pull request by rancher-max**
Hopefully this helps with flakiness...

### CODE REVIEW

**Comment by cbron** [comment]
Feel like this should at least be wrapped in a helper.

**Comment by rancher-max** [comment]
> Feel like this should at least be wrapped in a helper.

Yeah it probably should. If it works I will update

File: tests/testutil/service.go [hunk 1]
```diff
@@ -137,7 +137,13 @@ func (ts *TestService) Attach() []string {
 
 // Call "rio scale ns/service={scaleTo}"
 func (ts *TestService) Scale(scaleTo int) {
-	_, err := RioCmd([]string{"scale", fmt.Sprintf("%s=%d", ts.Name, scaleTo)})
+	var err error
+	for i := 0; i < 5; i++ {
```
Comment (StrongMonkey) [inline tests/testutil/service.go]: Also doing some backoff sleep would be better

**Comment by rancher-max** [comment]
Going to retry build a few times just to be sure this actually fixes the flaky tests, then we can merge

**Pull request merged**